packages/web-react/src/components/Select/useSelectStyleProps.ts

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import { Validation } from '../../types';
+import { useClassNamePrefix } from '../../hooks';
+
+export interface UseSelectStyleProps extends Validation {
+  isDisabled?: boolean;
+  isFluid?: boolean;
+  isLabelHidden?: boolean;
+}
+
+export interface useSelectStyleReturn {
```

Comment:
Please see most of the hooks.

```suggestion
export interface SelectStyles {
```

Reply:
So why is the incoming interface named the same as the hook, but not the outgoing one?
I'm working on the rules we should use. I use this suffix everywhere for outgoing interfaces